pydatajson/ckan_utils.py

```diff
@@ -117,18 +119,28 @@ def _get_theme_label(catalog, theme):
     return label
 
 
-def convert_iso_string_to_default_timezone(date_string):
+def convert_iso_string_to_default_timezone(date_string,
```

Comment:
Te quedó este nombre desactualizado, ya no es a un "to default timezone" sino a uno parametrizado

Reply:
Acepto sugerencias